### PR TITLE
Updated docs for randint operator

### PR DESCRIPTION
**Pull request by ChaiBapchya**
## Description ##
Updated the docs to reflect randint operator
Added for both NDArray and Symbol API (RandomNumberGenerator API)

### CODE REVIEW

**Comment by aaronmarkham** [comment]
Here's the current output at: http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13541/3/api/python/ndarray/ndarray.html#random-sampling



First thing I'm noticing is that you probably could have it showing minus `mxnet.ndarray` so each entry is like `random.x`. Note how the other sections are like this.

I see that `mxnet.random.seed` is different and not part of ndarray, so it make sense to have the full name.

Then within this `Random Sampling` category, the entries should be alphabetical. Rooting around in a long list for the function you want is difficult when it isn't alphabetical.

File: docs/api/python/ndarray/ndarray.md [hunk 1]
```diff
@@ -596,6 +596,7 @@ The `ndarray` package provides several classes:
     mxnet.ndarray.random.generalized_negative_binomial
     mxnet.ndarray.random.multinomial
     mxnet.ndarray.random.shuffle
+    mxnet.ndarray.random.randint
```
[the remark this answers is not in the document]
Comment (ChaiBapchya) [reply docs/api/python/ndarray/ndarray.md]: Done

File: docs/api/python/ndarray/random.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ In the rest of this document, we list routines provided by the `ndarray.random`
     uniform
     multinomial
     shuffle
+    randint
```
[the remark this answers is not in the document]
Comment (ChaiBapchya) [reply docs/api/python/ndarray/random.md]: Done

File: docs/api/python/symbol/symbol.md [hunk 3]
```diff
@@ -595,6 +595,7 @@ Composite multiple symbols into a new one by an operator.
     mxnet.symbol.random.generalized_negative_binomial
     mxnet.symbol.random.multinomial
     mxnet.symbol.random.shuffle
+    mxnet.symbol.random.randint
```
[the remark this answers is not in the document]
Comment (ChaiBapchya) [reply docs/api/python/symbol/symbol.md]: Done

File: docs/api/python/symbol/random.md [hunk 4]
```diff
@@ -37,6 +37,7 @@ In the rest of this document, we list routines provided by the `symbol.random` p
     uniform
     multinomial
     shuffle
+    randint
```
[the remark this answers is not in the document]
Comment (ChaiBapchya) [reply docs/api/python/symbol/random.md]: Done

File: docs/api/python/ndarray/random.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ChaiBapchya) [reply docs/api/python/ndarray/random.md]: Done

**Pull request merged**